cardano-node/docs/GenesisGeneration.md

```diff
@@ -0,0 +1,159 @@
+            "summand": "155381000000000"
+        },
+        "unlockStakeEpoch": "18446744073709551615",
+        "updateImplicit": "10000",
+        "updateProposalThd": "100000000000000",
+        "updateVoteThd": "1000000000000"
+    },
+    "bootStakeholders": {
+        "0d916567f96b6a65d204966e6aab5fbd242e56c321833f8ba5d607da": 1,
+        "4bd1884f5ce2231be8623ecf5778a9112e26514205b39ff53529e735": 1,
+        "5f53e01e1366aeda8811c2a630f0e037077a7b651093d2bdc4ef7200": 1,
+        "ada3ab5c69b945c33c15ca110b444aa58906bf01fcfe55d8818d9c49": 1
+    },
+    "ftsSeed": "736b6f766f726f64612047677572646120626f726f64612070726f766f646120",
+    "nonAvvmBalances": {},
+    "protocolConsts": {
+        "k": 2160,
+        "protocolMagic": 60987900,
+        "vssMaxTTL": 6,
+        "vssMinTTL": 2
+    },
+    "heavyDelegation": {
+        "c6c7fb227037a1719b9d871ea49b6039325aeb293915da7db9620e3f": {
+            "cert": "0f2ff9d1f071793dbd90fce8d47e96a09b594ee6dd00a4bac9664e4bd6af89830035ec921698b774c779eb1b6a2772d3d6ae37e630c06c75fbfecd02a6410a02",
+            "delegatePk": "I07+5HIUW0Lkq0poMMzGziuILwxSyTJ4lMSsoQz4HZunD5Xsx3MfBZWc4l+206lUOFaU+spdJg7MkmFKBoVV0g==",
+            "issuerPk": "AZzlT+pC5M4xG+GuRHJEXS+isikmVBorTqOyjRDGUQ+Lst9fn1zQn5OGKSXK29G2dn7R7JCugfcUebr0Dq7wPw==",
+            "omega": 1
+        }
+    },
+    "startTime": 1505621332,
+    "vssCerts": {
+        "0d916567f96b6a65d204966e6aab5fbd242e56c321833f8ba5d607da": {
+            "expiryEpoch": 1,
+            "signature": "396fe505f4287f832fd26c1eba1843a49f3d23b64d664fb3c8a2f25c8de73ce6f2f4cf37ec7fa0fee7750d1d6c55e1b07e1018ce0c6443bacdb01fb8e15f1a0f",
+            "signingKey": "ohsV3RtEFD1jeOzKwNulmMRhBG2RLdFxPbcSGbkmJ+xd/2cOALSDahPlydFRjd15sH0PkPE/zTvP4iN8wJr/hA==",
+            "vssKey": "WCECtpe8B/5XPefEhgg7X5veUIYH/RRcvXbz6w7MIJBwWYU="
+        },
+        ...
+    }
+}
+```
+
+#### FAQ:
+
+- What is an issuer? An issuer is a stakeholder/delegator that delegates their stake to a delegatee.
+
+### avvmDistr
+The key `avvmDistr` contains AVVM addresses with corresponding balances (lovelaces). People who purchased Ada at a pre-sale were issued a certificate at the Ada Voucher Vending Machine (AVVM) during the pre-sale period. AVVM addresses can be converted to Byron addresses and initially each address has this amount of ADA (as balance, not stake).
+
+### blockVersionData
+
+
+The key `blockVersionData` contains fundamental blockchain-related values:
+
+- `heavyDelThd` - heavyweight delegation threshold. The delegate (i.e stakeholder) must possess no less than this threshold of stake in order to participate in heavyweight delegation.
+- `maxBlockSize` - maximum size of block in bytes.
+- `maxHeaderSize` - maximum size of block's header in bytes.
+- `maxProposalSize` - maximum size of Cardano SL update proposal in bytes.
+- `maxTxSize` - maximum size of transaction in bytes.
+- `mpcThd` - threshold for participation in SSC (shared seed computation).
```

Comment:
Interestingly, this is an artifact from Ouroboros Classic, so the PBFT era has no use for this value.

@dcoutts might clarify this further.

Reply:
I saw on slack that we were dropping the `SSC` stuff. If that is the case I can document that here. Maybe I should also change the title to `Byron/PBFT era genesis data format`. Thoughts @deepfire ?

Reply:
@Jimbo4350, it's an odd situation.. the genesis is sealed for `mainnet`, however its interpretation is changing.

I'd say we want to explain the currently relevant picture of things, and if so, it follows that we must explain the effective delta in the interpretation.

And if so changing the title wouldn't be useful.